src/actions/markdownActions.test.js

```diff
@@ -0,0 +1,13 @@
+import { updateMarkdown, UPDATE_MARKDOWN } from './markdownActions';
+
+
+describe('Update markdown action', () => {
```

Comment:
I would name this something that will scale better as you add more actions. Maybe `markdown actions`?

Reply:
Hello,

I have made changes to this PR based on your comments. 
Thank you,
Lance